runtime/pack/dist/opt/cfilter/plugin/cfilter.lua

```diff
@@ -90,7 +91,7 @@ Qf_filter = function(qf, searchpat, bang)
   end
 
   items = vim9.fn_mut('filter', { Xgetlist(), Cond }, { replace = 0 })
-  title = cmd .. ' /' .. pat .. '/'
+  title = string.format([=[{cmd} /{pat}/]=])
```

Comment:
?

Reply:
That is what vim9jit generated.

Reply:
Did you regenerate the files in this PR, or why was this changed?

Reply:
Ah, vim9jit is broken then

Reply:
> Did you regenerate the files in this PR, or why was this changed?

I did at first, but I reverted it (except I kept the lines that disabled lints)